js/screens/CommunityQuestion/CommunityQuestion.js

```diff
@@ -18,19 +19,27 @@ const CommunityQuestion = props => {
       style={styles.linearGradient}
     >
       <ScrollView>
-        <View style={styles.container}>
+        <View style={props.isVisible ? styles.dimContainer : styles.container}>
           <Text style={styles.title}>Ask a question</Text>
           <LinearGradient
             start={{ x: 1, y: 0 }}
             end={{ x: 1, y: 1 }}
             colors={["#FAD961", "#EFCC4C"]}
             style={styles.crisisButton}
           >
-            <TouchableOpacity>
+            <TouchableOpacity
+              onPress={() => {
+                props.toggleVisibility();
+              }}
+            >
               <Text style={styles.crisisText}>I'm in a crisis</Text>
             </TouchableOpacity>
           </LinearGradient>
 
+          <CrisisModal
+            toggleVisibility={props.toggleVisibility}
+            isVisible={props.isVisible}
+          />
           <CommunityQuestionForm navigation={props.navigation} />
         </View>
       </ScrollView>
```

Comment:
you are receiving more props than just a navigation object. Please add prop type validation for all of your props.